_sass/_custom_classes.scss

```diff
@@ -138,18 +138,7 @@ $card-slide-transition:
       position: relative;  
       border-top: 1px solid $primary;
       border-bottom: 1px solid $primary;
-    }
-
-    .featured-content::before {    
```

Comment:
This `::before` element was preventing the "See all use cases" link from being clickable, so I changed it to a flat background colour